src/db/migrations/20190821154724-create-trip.js

```diff
@@ -36,10 +36,6 @@ module.exports = {
       return_date: {
         type: Sequelize.DATE // return date allows null because it could be a one-way-trip
       },
-      reason: {
```

Comment:
Could you please explain why 'reason' was removed?

Reply:
`Trips` should not have `reasons`. That is why the PT asked for a `Reqest Table`. It is more appropriate to have a `reason` plus `status` on the request page. Since the outcome of a trip will be decided by if the request will be approved or not; based on your `reasons`

Reply:
What would have changed if you had renamed `trips` to `requests`.
Again combine the columns you have and see if there is any difference.

Reply:
**Copied**: Trips are the actual planned movement from point A to B. Requests are the reasons and plea for such trips to happen. Two distinct models. Plus ideally, A request should be granted, before a trip can be made. A rejected request nullifies a trip. You can choose to nullify the whole trip row if it were integrated as one. Or you can choose to nullify just the request row if it were abstracted. Request table = lesser columns, Trips table -= more columns. Speed vs Compact. Speed wins all the time.